applications/surgical_scene_recon/run_surgical_recon.py

```
@@ -0,0 +1,265 @@
+        "--checkpoint",
+        args.checkpoint,
+        "--num_frames",
+        str(args.num_frames),
+    ]
+
+    if not args.loop:
+        cmd.append("--no-loop")
+
+    print(f"Running: {' '.join(cmd)}\n")
+
+    return subprocess.call(cmd)
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)